### PR TITLE
Add async API for netstandard2.0.

### PR DESCRIPTION
**Pull request by mo-esmp**
Add async API for netstandard2.0 projects without changing project structure and touching sync APIs and also not visible for full .NET Framework projects.

### CODE REVIEW

**Comment by mo-esmp** [comment]
Are you going to take action?

**Comment by hosseinRezaei188** [comment]
> Are you going to take action?
Tank you for contribution, we are reviewing your code.

File: Kavenegar/KavenegarApi.cs [hunk 1]
```diff
@@ -641,5 +663,521 @@ public List<SendResult> CallMakeTTS(string message, List<string> receptor, DateT
 
         #endregion << CallMakeTTS >>
 
+#if NETSTANDARD2_0
+
+        public SendResult SendAsync(string sender, string receptor, string message)
+        {
+            return Send(sender, receptor, message, MessageType.MobileMemory, DateTime.MinValue);
```
Comment (hosseinRezaei188) [inline Kavenegar/KavenegarApi.cs]: why dosent use async send method?

**Comment by mo-esmp** [comment]
Ops copy past mistake. I will fixt it.

File: Kavenegar/KavenegarApi.cs (hunk 1 above)
Comment (mo-esmp) [reply Kavenegar/KavenegarApi.cs]: Ops copy past mistake. Now it should be fixed.

Comment (hosseinRezaei188) [reply Kavenegar/KavenegarApi.cs]: thank you bro

**Pull request merged**